gems/pending/appliance_console/utilities.rb

```diff
@@ -32,7 +32,7 @@ def self.bail_if_db_connections(message)
     def self.db_host_database_region
       result = AwesomeSpawn.run(
         "bin/rails runner",
-        :params => ["puts MiqDbConfig.current.options.values_at(:host, :database), ActiveRecord::Base.my_region_number"],
+        :params => ["puts MiqDbConfig.current.options.values_at(:host, :database), ApplicationRecord.my_region_number"],
```

Comment:
:+1: thanks.
When I made this original change, I knew this future change would have an effect. Sorry that I didn't put in an issue to track it.

Thanks for circling back.